docs/lakehouse/datalake-analytics/iceberg.md

```diff
@@ -31,6 +31,7 @@ under the License.
 
 1. Support Iceberg V1/V2.
 2. The V2 format only supports Position Delete, not Equality Delete.
+3. Supports Parquet format. And ORC format since 2.1.3.
```

Comment:
3. Support Parquet and ORC format since 2.1.3.

Reply:
Already support Parquet. ORC is since 2.1.3.
I will separate them in 2 lines

Reply:
done